Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -893,81 +916,79 @@ - (instancetype)initWithParentAdapter:(ALVungleMediationAdapter *)parentAdapter
+                        withObject: extraInfo];
 }
 
-- (void)bannerAdDidFailToLoad:(VungleBanner *)banner withError:(NSError *)error
+- (void)bannerAdDidFail:(VungleBannerView *)bannerView withError:(NSError *)error
 {
     MAAdapterError *adapterError = [ALVungleMediationAdapter toMaxError: error isAdPresentError: NO];
     [self.parentAdapter log: @"AdView failed to load with error: %@", adapterError];
     [self.delegate didFailToLoadAdViewAdWithError: adapterError];
 }
 
-- (void)bannerAdWillPresent:(VungleBanner *)banner
+- (void)bannerAdWillPresent:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad will present %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad will present %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidPresent:(VungleBanner *)banner
+- (void)bannerAdDidPresent:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad shown %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad shown %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidTrackImpression:(VungleBanner *)banner
+- (void)bannerAdDidTrackImpression:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad impression tracked %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad impression tracked %@", bannerView.placementId];
     [self.delegate didDisplayAdViewAd];
 }
 
-- (void)bannerAdDidClick:(VungleBanner *)banner
+- (void)bannerAdDidClick:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad clicked %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad clicked %@", bannerView.placementId];
     [self.delegate didClickAdViewAd];
 }
 
-- (void)bannerAdWillLeaveApplication:(VungleBanner *)banner
+- (void)bannerAdWillLeaveApplication:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad will leave application %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad will leave application %@", bannerView.placementId];
 }
 
-- (void)bannerAdDidFailToPresent:(VungleBanner *)banner withError:(NSError *)error
+- (void)bannerAdWillClose:(VungleBannerView *)bannerView
 {
-    MAAdapterError *adapterError = [ALVungleMediationAdapter toMaxError: error isAdPresentError: YES];
-    [self.parentAdapter log: @"AdView ad failed to present with error: %@", adapterError];
-    [self.delegate didFailToDisplayAdViewAdWithError: adapterError];
+    [self.parentAdapter log: @"AdView ad will close %@", bannerView.placementId];
 }
 
-- (void)bannerAdWillClose:(VungleBanner *)banner
+- (void)bannerAdDidClose:(VungleBannerView *)bannerView
 {
-    [self.parentAdapter log: @"AdView ad will close %@", banner.placementId];
+    [self.parentAdapter log: @"AdView ad hidden %@", bannerView.placementId];
+    [self.delegate didHideAdViewAd];
 }
 
-- (void)bannerAdDidClose:(VungleBanner *)banner
+- (void)adViewWillChangeToSize:(VungleBannerView * _Nonnull)bannerView :(VungleAdSize * _Nonnull)size
```

Comment:
Do we need this? Max may concern about this since we call didLoadAdForAdView with bannerView when ad loaded, actually SDK already resized the bannerView.

Reply:
Probably not. I will remove this since I can access to the "final ad size from VungleBannerView object now.

Reply:
Removed it.